release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1163,17 +1163,21 @@ The `haproxy.router.openshift.io/balance` variable, which sets the router load-b
+In previous releases, two service account token secrets were generated when a service account was created:
 
-If you need a service account token secret, you must manually use the TokenRequest API to request bound service account tokens or create a service account token secret.
+* A service account token secret for authenticating to the internal {product-title} registry
+* A service account token secret for accessing the Kubernetes API
 
-After updating to {product-version}, existing service account token secrets are not deleted and continue to function as expected.
+Starting with {product-title} 4.11, this second service account token secret for accessing the Kubernetes API is no longer created. This is because the `LegacyServiceAccountTokenNoAutoGeneration` upstream Kubernetes feature gate was enabled in Kubernetes 1.24, which stops the automatic generation of secret-based service account tokens for accessing the Kubernetes API. After upgrading to {product-title} 4.11, any existing service account token secrets are not deleted and continue to function.
 
-Service Account token secrets still appear as auto-generated in {product-title} 4.11. However, instead of two secrets per Service Account, there will now be only one, which will be further reduced to zero in a future release. These tokens do not work. For now, dockercfg secrets are still being generated as secrets and no secrets will be deleted during upgrades.
+[NOTE]
+====
+Do not rely on these automatically generated secrets for your own use; they might be removed in a future {product-title} release.
+====
 
-* For information about using the TokenRequest API, see xref:../authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens[Using bound service account tokens]
+If your workload needs a service account token, use the `TokenRequest` API to request a bound service account token. Tokens obtained from the `TokenRequest` API are more secure than service account token secrets, because they have a bounded lifetime and are not readable by other API clients. For more information, see xref:../authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens[Using bound service account tokens].
 
-* For information about creating a service account token secret, see xref:../nodes/pods/nodes-pods-secrets.html#nodes-pods-secrets-creating-sa_nodes-pods-secrets[Creating a service account token secret].
+You can also manually create a service account token secret to create a token, if the security exposure of a non-expiring token in a readable API object is acceptable to you. For more information, see xref:../nodes/pods/nodes-pods-secrets.html#nodes-pods-secrets-creating-sa_nodes-pods-secrets[Creating a service account token secret].
```

Comment:
nit: stylistically I'd say "create ... to obtain a token" would work better

Reply:
Good catch, agreed